### ISSUE TITLE
Test report publishing fails following v3.3.0 release

### ISSUE THREAD

**Issue by bogdanbranisteanu** [mikepenz/action-junit-report#637]
Hi, I have been using your test report publishing scripts in several projects. Today you released a new version and since then the publishing fails.

<img width="1023" alt="image" src="https://user-images.githubusercontent.com/108867575/185598739-feb2d7a9-0212-48b2-82be-2b8d5e266002.png">

It seems to have something to do with the summary input, but this input is described as optional in the documentation.

**Comment by mikepenz** [comment]
@bogdanbranisteanu thank you for the report. Are you using the specific version `@v3.3.0` or the higher level tag `@v3`?

Could be related to: https://github.com/mikepenz/action-junit-report/blob/main/src/annotator.ts#L140-L143
But the summary was already in use on v3.1.0 too https://github.com/mikepenz/action-junit-report/blob/v3.1.0/src/main.ts#L181 🤔 

Is this on an open source repo? E.g. GitHub Cloud or enterprise server?

**Comment by bogdanbranisteanu** [comment]
Tried with both @v3.3.0 and @v3 and it was failing. Now changed to @v3.2.0 and it works just fine. The projects are on an enterprise server.

**Comment by mikepenz** [comment]
`@v3` is tagged now on the same level as `@v3.3.0` (always verifying this though) 

Wonder if it may be due to the upgrade to `core@1.9.1`

**Comment by mikepenz** [comment]
I've recompiled `v3.3.0` with the previous `core@1.9.0` version here: https://github.com/mikepenz/action-junit-report/commit/c3ef56bd1c1d7f5a1e8ee7e561369b24efed4ed8

Would you please be so kind to test if that is the cause of the problem. 

```
mikepenz/action-junit-report@c3ef56bd1c1d7f5a1e8ee7e561369b24efed4ed8
```
(generally speaking commit hash pinning is from a security perspective advised - https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions)

**Comment by bogdanbranisteanu** [comment]
It's still failing.

**Comment by mikepenz** [comment]
@bogdanbranisteanu thank you for trying this variant. 

What seems strange, it should fail also on v3.2.0 given that it also posts a job summary. 

-----

I am sorry that I have to rely on you testing this - Really appreciate your help.

Prepared a new commit which allows to disable the publishing of the job summary at the end: 
https://github.com/mikepenz/action-junit-report/commit/2e7e69d112175cca18d6fa2b53566862c24a9f20

```
mikepenz/action-junit-report@2e7e69d112175cca18d6fa2b53566862c24a9f20
```

```
      - uses: mikepenz/action-junit-report@2e7e69d112175cca18d6fa2b53566862c24a9f20
        with:
          job_summary: false
```

**Comment by bogdanbranisteanu** [comment]
This one works fine.

**Comment by mikepenz** [comment]
Published v3.3.1 which offers the APi to disable `job_summary`'s.

**Comment by bogdanbranisteanu** [comment]
I can confirm that all projects can now successfully publish the test reports. Thanks a lot for the cooperation and quick fix!
